website/docs/docs/collaborate/manage-access/audit-log.md

```diff
@@ -9,7 +9,7 @@ To review actions performed by people in your organization, dbt provides logs of
 
 :::note
 
-Audit logs are currently not supported for Microsoft Azure.
+Audit logs are currently not supported for single-tenant customers hosted on Microsoft Azure.
```

Comment:
@will-sargent-dbtlabs thank you for submitting this change! One minor suggestion. Can we make the sentence active and link to ST definition?

```suggestion
Single-tenant deployment environments hosted on Microsoft Azure do not currently support audit logs. For more information, refer to [Single tenant](/docs/deploy/single-tenant).
```

Reply:
Sounds great @runleonarun ! Thanks